dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java

```diff
@@ -77,12 +78,21 @@ public void setRestEndpoint(String restEndpoint) {
         this.restEndpoint = restEndpoint;
     }
 
+    public String getproductionNoteDirectory() {
+        return productionNoteDirectory;
+    }
+
+    public void setproductionNoteDirectory(String productionNoteDirectory) {
+        this.productionNoteDirectory = productionNoteDirectory;
+    }
+
     @Override
     public String toString() {
         return "ZeppelinParameters{"
                + "noteId='" + noteId + '\''
-               + ", paragraphId='" + paragraphId + '\''
+               + ", paragraphId='" + paragraphId + '\'' 
```

Comment:
Can you help to also refactor this class to use lombok annotations?

Reply:
@kezhenxu94 Certainly yes! I'm a lombok `fan`, lol.